src/playercards/cards/WellConnected.ttslua

```diff
@@ -0,0 +1,70 @@
+-- this script is shared between both the level 0 and the upgraded level 3 version of the card
+local playmatApi = require("playermat/PlaymatApi")
+
+local display = false
+local count = 0
+local modValue = 5 -- level 0 Well Connected
+local loopId = nil
+
+local b_display = {
+    click_function = "toggleCounter",
+    function_owner = self,
+    position = {0.88,0.5,-1.33},
+    font_size = 150,
+    width = 175,
+    height = 175
+}
+
+function onLoad(saved_data)
+    local notes = JSON.decode(self.getGMNotes())
+
+    if notes.id == "54006" then -- hardcoded card id for upgraded Well Connected (3)
+        modValue = 4 -- Well Connected (3)
+    end
+
+    if saved_data != '' then
+        local loaded_data = JSON.decode(saved_data)
+        display = not loaded_data.saved_display
+
+        toggleCounter()
+    end
```

Comment:
This causes an error if you save & load when the display is not enabled since it does not have a loopId defined, but tries to stop the loop